http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcHttpFacade.java

```diff
@@ -26,11 +26,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.io.UnsupportedEncodingException;
-import java.net.InetAddress;
-import java.net.InetSocketAddress;
-import java.net.URI;
-import java.net.URLDecoder;
+import java.net.*;
```

Comment:
Hello @kanha-gupta , thank you for the PR! Just some minor here. You can try to build wildfly-elytron project by `mvn clean install` in its parent folder. If you have problems you can try to look at our contribution guide and see how to setup your environment https://wildfly-security.github.io/wildfly-elytron/contribute/ . Let us know if you have issues.

On this line, if you try to run run `mvn clean install` you will see it fail because of checkstyle. We have a rule that asterisk in import cannot be used.